content/notebooks/pandeia/pandeia.ipynb

```diff
@@ -334,9 +335,9 @@
    "metadata": {},
    "source": [
     "#### Step 2: Determining Optimal Number of Exposures\n",
-    "With the magnitude determined, we then calculate the optimal number of exposures required to achieve a specified SNR for a known flux. This is done by simulating observations with varying numbers of exposures, identifying the minimum exposure count necessary to meet or exceed the target SNR. This ensures efficient use of telescope time while maintaining data quality.\n",
+    "Once the magnitude is determined, we calculate the optimal number of exposures needed to achieve a target SNR for a known flux. This involves simulating observations with varying exposure counts and identifying the minimum number required to meet or exceed the target SNR. This approach ensures efficient use of telescope time while maintaining data quality.\n",
     "\n",
-    "The following helper functions use Pandeia to simulate a range of scenes with different numbers of exposures in order to estimate the optimal observing time to reach the expected limiting magnitude for a source with a given flux. As above, we assume a point source with a flat spetrum, and the MA table is set to the \"c2a_img_hlwas\" table, truncated to 8 resultants.\n",
+    "The following helper functions use Pandeia to simulate a range of scenes with different number of exposures in order to estimate the optimal observing time to reach the expected limiting magnitude for a source with a given flux. As above, we assume a point source with a flat spetrum, and the MA table is set to the \"c2a_img_hlwas\" table, truncated to 8 resultants.\n",
```

Comment:
The word "number" should be plural here. It should read "...a range of scenes with different numbers of exposures..."